Dockerfile

```diff
@@ -1,5 +1,4 @@
-ARG BASE_IMAGE
-FROM ${BASE_IMAGE}
+FROM container-registry.zalando.net/library/eclipse-temurin-11-jre:latest
```

Comment:
It makes sense to base on JDK, as sometimes we need to take a memory dump, and with JDK installed it is much simpler